src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -406,6 +406,16 @@ class genUtils {
         return inputNumber.toLocaleString(undefined, { minimumFractionDigits: n, maximumFractionDigits: n });
     };
 
+    /***  File Methods ***/
+    
+    static isFileOfType(fileName: string, possibleFileTypes: string[]): boolean {
```

Comment:
I would use even path here - as it should well for any path, not only fileName ;-)

Reply:
done - using getFileExtension method